e2e/libs/page-events-helper.ts

```diff
@@ -44,7 +44,8 @@ const isWorkbenchRequest = (request: Request): Request | null => {
   ];
   return request &&
     !unwantedRequests.some((urlPart) => request.url().includes(urlPart)) &&
-    /all-of-us-workbench-(test|staging).appspot.com/.exec(request.url()) != null
```

Comment:
`all-of-us-workbench-staging.appspot.com` is not an valid Workbench app URL.

Reply:
Hmm.. this is already missing perf. If we ever run this on higher environments, it will be pretty easy to miss updating this code. I would recommend moving this into the workbench-config settings; this way it will be easily updated when we change environments.

I would also suggest you make the target more specific and name the property something like "apiHostname": `api-dot-all-of-us-workbench-test.appspot.com`

Reply:
Also.. isn't localhost missing here too?

Reply:
Very good point! Created new `API_HOSTNAME` config. Removed `API_URL` b/c it was not used.